### PR TITLE
A small change and new gene

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Makes it so Cat Eyes now give an additional 1 more darksight
New Gene found in bats, birds and hell pigs, Blood bank
Blood banks makes you regenerate 60 additonal blood and storage of said blood
Cat Eyes now also work

### CODE REVIEW

**Comment by Memedoktor** [comment]
so to be clear this makes it so your max bloodvolume goes up by 60 and you regenerate to that new max?

**Comment by Trilbyspaceclone** [comment]
Yes

**Review by hex37 — APPROVED** [review]

**Comment by Dimasw99** [comment]
Getting twice the stun duration and eye damage for just +1 dark sight? Is it even worth it then? To me it sounds heavily under powered.

File: code/modules/genetics/mutations/blood_bank.dm [hunk 1]
```diff
@@ -0,0 +1,22 @@
+//Macro: MUTATION_BLOOD_BANK
+/datum/genetics/mutation/blood_bank
+	name = "Blood Banks"
+	key = "MUTATION_BLOOD_BANK"
+	desc = "Slightly increases the amount of blood the person has."
+	gain_text = "Your eyes narrow into that of a predator."
```
Comment (Dimasw99) [inline code/modules/genetics/mutations/blood_bank.dm:6]:
```suggestion
	gain_text = "You feel your veins tingle and pulsate."
```

**Pull request merged**